content/about/feature-stages.md

```diff
@@ -70,33 +75,34 @@ Below is our list of existing features and their current phases. This informatio
 | [Pluggable Key/Cert Support for Istio CA](/docs/tasks/security/plugin-ca-cert/)        | Stable
 | [Service-to-service mutual TLS](/docs/concepts/security/mutual-tls/)         | Stable
 | [Authentication policy](/docs/concepts/security/authn-policy/)  | Alpha
-| [VM: Service Credential Distribution](/docs/concepts/security/mutual-tls/)         | Alpha
+| [VM: Service Credential Distribution](/docs/concepts/security/mutual-tls/)         | Beta
 | [OPA Checker](https://github.com/istio/istio/blob/{{<branch_name>}}/mixer/adapter/opa/README.md)    | Alpha
-| RBAC Mixer Adapter     | Alpha
-| [API Keys](/docs/reference/config/policy-and-telemetry/templates/apikey/) | Alpha
+| [RBAC Mixer Adapter](https://istio.io/docs/concepts/security/rbac/)   | Alpha
+
 
 ### Core
 
 | Feature           | Phase
 |-------------------|-------------------
 | [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Beta
-| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Beta
+| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Alpha
+| [Kubernetes: Istio Control Plane Upgrade](/docs/setup/kubernetes/) | Beta
```

Comment:
Why is upgrade beta but installation alpha? Seems like installation should be beta then?

Reply:
You are right, changed installation to beta